### ISSUE TITLE
Chess piece as wrong color when promoting

### ISSUE THREAD

**Issue by lucalaissue** [HE-Arc/AnalyChess#28]
When parsing a PGN, when there's a promoting move, the promoted piece has the wrong color.

For example with this pgn
```
1. d4 d5 2. c4 dxc4 3. e4 Nc6 4. Be3 Nf6 5. Nc3 e5 6. d5 Ne7 7. Bxc4 Ng6 8. f3 Bd6 9. Qd2 Bd7 10. Nge2 a6 11. Bb3 b5
12. a4 O-O 13. O-O Qe7 14. Rac1 Nh5 15. g3 h6 16. Bc2 Rab8 17. axb5 axb5 18. Ra1 Ra8 19. Bd3 Bb4 20. Rxa8 Rxa8
21. Qc2 Bc5 22. Nd1 Bd6 23. Nf2 Nhf4 24. Rc1 Qg5 25. Kh1 Qh5 26. Ng1 Nxd3 27. Nxd3 f5 28. Nc5 Bc8 29. Rf1 Ne7
30. Qd3 fxe4 31. fxe4 Qg6 32. Kg2 Kh7 33. Nf3 Ng8 34. Nh4 Qg4 35. Nf5 Nf6 36. h3 Qg6 37. Ne6 Ra4 38. b3 Rxe4
39. Nxd6 Bxe6 40. dxe6 cxd6 41. e7 d5 42. Bc5 Qe8 43. Qf3 Qc6 44. b4 Qe8 45. Qf5+ Kh8 46. Qxf6 gxf6 47. Rxf6 Qh5
48. Rf8+ Kg7 49. e8=Q Re2+ 50. Kf1 Qxh3+ 51. Kxe2 Qg2+ 52. Rf2 Qe4+ 53. Kd2
```
once the PGN is parsed, on white's move number 49, the generated piece is black in the analysis file :

```json
{"title":"Title","description":"Description","result":"Result","moves":[{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":3,"sRow":1,"eFile":3,"eRow":3,"mPiece":""}],"pgnMove":"d4"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":3,"sRow":6,"eFile":3,"eRow":4,"mPiece":""}],"pgnMove":"d5"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":2,"sRow":1,"eFile":2,"eRow":3,"mPiece":""}],"pgnMove":"c4"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":3,"sRow":4,"eFile":2,"eRow":3,"mPiece":""}],"pgnMove":"dxc4"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":4,"sRow":1,"eFile":4,"eRow":3,"mPiece":""}],"pgnMove":"e4"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":1,"sRow":7,"eFile":2,"eRow":5,"mPiece":""}],"pgnMove":"Nc6"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":2,"sRow":0,"eFile":4,"eRow":2,"mPiece":""}],"pgnMove":"Be3"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":6,"sRow":7,"eFile":5,"eRow":5,"mPiece":""}],"pgnMove":"Nf6"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":1,"sRow":0,"eFile":2,"eRow":2,"mPiece":""}],"pgnMove":"Nc3"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":4,"sRow":6,"eFile":4,"eRow":4,"mPiece":""}],"pgnMove":"e5"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":3,"sRow":3,"eFile":3,"eRow":4,"mPiece":""}],"pgnMove":"d5"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":2,"sRow":5,"eFile":4,"eRow":6,"mPiece":""}],"pgnMove":"Ne7"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":5,"sRow":0,"eFile":2,"eRow":3,"mPiece":""}],"pgnMove":"Bxc4"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":4,"sRow":6,"eFile":6,"eRow":5,"mPiece":""}],"pgnMove":"Ng6"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":5,"sRow":1,"eFile":5,"eRow":2,"mPiece":""}],"pgnMove":"f3"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":5,"sRow":7,"eFile":3,"eRow":5,"mPiece":""}],"pgnMove":"Bd6"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":3,"sRow":0,"eFile":3,"eRow":1,"mPiece":""}],"pgnMove":"Qd2"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":2,"sRow":7,"eFile":3,"eRow":6,"mPiece":""}],"pgnMove":"Bd7"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":6,"sRow":0,"eFile":4,"eRow":1,"mPiece":""}],"pgnMove":"Nge2"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":0,"sRow":6,"eFile":0,"eRow":5,"mPiece":""}],"pgnMove":"a6"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":2,"sRow":3,"eFile":1,"eRow":2,"mPiece":""}],"pgnMove":"Bb3"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":1,"sRow":6,"eFile":1,"eRow":4,"mPiece":""}],"pgnMove":"b5"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":0,"sRow":1,"eFile":0,"eRow":3,"mPiece":""}],"pgnMove":"a4"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":4,"sRow":7,"eFile":6,"eRow":7,"mPiece":""},{"sFile":7,"sRow":7,"eFile":5,"eRow":7,"mPiece":""}],"pgnMove":"O-O"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":4,"sRow":0,"eFile":6,"eRow":0,"mPiece":""},{"sFile":7,"sRow":0,"eFile":5,"eRow":0,"mPiece":""}],"pgnMove":"O-O"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":3,"sRow":7,"eFile":4,"eRow":6,"mPiece":""}],"pgnMove":"Qe7"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":0,"sRow":0,"eFile":2,"eRow":0,"mPiece":""}],"pgnMove":"Rac1"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":5,"sRow":5,"eFile":7,"eRow":4,"mPiece":""}],"pgnMove":"Nh5"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":6,"sRow":1,"eFile":6,"eRow":2,"mPiece":""}],"pgnMove":"g3"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":7,"sRow":6,"eFile":7,"eRow":5,"mPiece":""}],"pgnMove":"h6"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":1,"sRow":2,"eFile":2,"eRow":1,"mPiece":""}],"pgnMove":"Bc2"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":0,"sRow":7,"eFile":1,"eRow":7,"mPiece":""}],"pgnMove":"Rab8"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":0,"sRow":3,"eFile":1,"eRow":4,"mPiece":""}],"pgnMove":"axb5"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":0,"sRow":5,"eFile":1,"eRow":4,"mPiece":""}],"pgnMove":"axb5"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":2,"sRow":0,"eFile":0,"eRow":0,"mPiece":""}],"pgnMove":"Ra1"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":1,"sRow":7,"eFile":0,"eRow":7,"mPiece":""}],"pgnMove":"Ra8"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":2,"sRow":1,"eFile":3,"eRow":2,"mPiece":""}],"pgnMove":"Bd3"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":3,"sRow":5,"eFile":1,"eRow":3,"mPiece":""}],"pgnMove":"Bb4"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":0,"sRow":0,"eFile":0,"eRow":7,"mPiece":""}],"pgnMove":"Rxa8"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":5,"sRow":7,"eFile":0,"eRow":7,"mPiece":""}],"pgnMove":"Rxa8"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":3,"sRow":1,"eFile":2,"eRow":1,"mPiece":""}],"pgnMove":"Qc2"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":1,"sRow":3,"eFile":2,"eRow":4,"mPiece":""}],"pgnMove":"Bc5"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":2,"sRow":2,"eFile":3,"eRow":0,"mPiece":""}],"pgnMove":"Nd1"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":2,"sRow":4,"eFile":3,"eRow":5,"mPiece":""}],"pgnMove":"Bd6"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":3,"sRow":0,"eFile":5,"eRow":1,"mPiece":""}],"pgnMove":"Nf2"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":7,"sRow":4,"eFile":5,"eRow":3,"mPiece":""}],"pgnMove":"Nhf4"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":5,"sRow":0,"eFile":2,"eRow":0,"mPiece":""}],"pgnMove":"Rc1"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":4,"sRow":6,"eFile":6,"eRow":4,"mPiece":""}],"pgnMove":"Qg5"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":6,"sRow":0,"eFile":7,"eRow":0,"mPiece":""}],"pgnMove":"Kh1"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":6,"sRow":4,"eFile":7,"eRow":4,"mPiece":""}],"pgnMove":"Qh5"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":4,"sRow":1,"eFile":6,"eRow":0,"mPiece":""}],"pgnMove":"Ng1"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":5,"sRow":3,"eFile":3,"eRow":2,"mPiece":""}],"pgnMove":"Nxd3"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":5,"sRow":1,"eFile":3,"eRow":2,"mPiece":""}],"pgnMove":"Nxd3"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":5,"sRow":6,"eFile":5,"eRow":4,"mPiece":""}],"pgnMove":"f5"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":3,"sRow":2,"eFile":2,"eRow":4,"mPiece":""}],"pgnMove":"Nc5"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":3,"sRow":6,"eFile":2,"eRow":7,"mPiece":""}],"pgnMove":"Bc8"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":2,"sRow":0,"eFile":5,"eRow":0,"mPiece":""}],"pgnMove":"Rf1"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":6,"sRow":5,"eFile":4,"eRow":6,"mPiece":""}],"pgnMove":"Ne7"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":2,"sRow":1,"eFile":3,"eRow":2,"mPiece":""}],"pgnMove":"Qd3"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":5,"sRow":4,"eFile":4,"eRow":3,"mPiece":""}],"pgnMove":"fxe4"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":5,"sRow":2,"eFile":4,"eRow":3,"mPiece":""}],"pgnMove":"fxe4"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":7,"sRow":4,"eFile":6,"eRow":5,"mPiece":""}],"pgnMove":"Qg6"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":7,"sRow":0,"eFile":6,"eRow":1,"mPiece":""}],"pgnMove":"Kg2"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":6,"sRow":7,"eFile":7,"eRow":6,"mPiece":""}],"pgnMove":"Kh7"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":6,"sRow":0,"eFile":5,"eRow":2,"mPiece":""}],"pgnMove":"Nf3"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":4,"sRow":6,"eFile":6,"eRow":7,"mPiece":""}],"pgnMove":"Ng8"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":5,"sRow":2,"eFile":7,"eRow":3,"mPiece":""}],"pgnMove":"Nh4"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":6,"sRow":5,"eFile":6,"eRow":3,"mPiece":""}],"pgnMove":"Qg4"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":7,"sRow":3,"eFile":5,"eRow":4,"mPiece":""}],"pgnMove":"Nf5"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":6,"sRow":7,"eFile":5,"eRow":5,"mPiece":""}],"pgnMove":"Nf6"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":7,"sRow":1,"eFile":7,"eRow":2,"mPiece":""}],"pgnMove":"h3"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":6,"sRow":3,"eFile":6,"eRow":5,"mPiece":""}],"pgnMove":"Qg6"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":2,"sRow":4,"eFile":4,"eRow":5,"mPiece":""}],"pgnMove":"Ne6"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":0,"sRow":7,"eFile":0,"eRow":3,"mPiece":""}],"pgnMove":"Ra4"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":1,"sRow":1,"eFile":1,"eRow":2,"mPiece":""}],"pgnMove":"b3"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":0,"sRow":3,"eFile":4,"eRow":3,"mPiece":""}],"pgnMove":"Rxe4"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":5,"sRow":4,"eFile":3,"eRow":5,"mPiece":""}],"pgnMove":"Nxd6"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":2,"sRow":7,"eFile":4,"eRow":5,"mPiece":""}],"pgnMove":"Bxe6"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":3,"sRow":4,"eFile":4,"eRow":5,"mPiece":""}],"pgnMove":"dxe6"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":2,"sRow":6,"eFile":3,"eRow":5,"mPiece":""}],"pgnMove":"cxd6"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":4,"sRow":5,"eFile":4,"eRow":6,"mPiece":""}],"pgnMove":"e7"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":3,"sRow":5,"eFile":3,"eRow":4,"mPiece":""}],"pgnMove":"d5"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":4,"sRow":2,"eFile":2,"eRow":4,"mPiece":""}],"pgnMove":"Bc5"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":6,"sRow":5,"eFile":4,"eRow":7,"mPiece":""}],"pgnMove":"Qe8"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":3,"sRow":2,"eFile":5,"eRow":2,"mPiece":""}],"pgnMove":"Qf3"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":4,"sRow":7,"eFile":2,"eRow":5,"mPiece":""}],"pgnMove":"Qc6"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":1,"sRow":2,"eFile":1,"eRow":3,"mPiece":""}],"pgnMove":"b4"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":2,"sRow":5,"eFile":4,"eRow":7,"mPiece":""}],"pgnMove":"Qe8"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":5,"sRow":2,"eFile":5,"eRow":4,"mPiece":""}],"pgnMove":"Qf5+"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":7,"sRow":6,"eFile":7,"eRow":7,"mPiece":""}],"pgnMove":"Kh8"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":5,"sRow":4,"eFile":5,"eRow":5,"mPiece":""}],"pgnMove":"Qxf6"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":6,"sRow":6,"eFile":5,"eRow":5,"mPiece":""}],"pgnMove":"gxf6"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":5,"sRow":0,"eFile":5,"eRow":5,"mPiece":""}],"pgnMove":"Rxf6"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":4,"sRow":7,"eFile":7,"eRow":4,"mPiece":""}],"pgnMove":"Qh5"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":5,"sRow":5,"eFile":5,"eRow":7,"mPiece":""}],"pgnMove":"Rf8+"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":7,"sRow":7,"eFile":6,"eRow":6,"mPiece":""}],"pgnMove":"Kg7"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":4,"sRow":6,"eFile":4,"eRow":7,"mPiece":"q"}],"pgnMove":"e8=Q"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":4,"sRow":3,"eFile":4,"eRow":1,"mPiece":""}],"pgnMove":"Re2+"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":6,"sRow":1,"eFile":5,"eRow":0,"mPiece":""}],"pgnMove":"Kf1"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":7,"sRow":4,"eFile":7,"eRow":2,"mPiece":""}],"pgnMove":"Qxh3+"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":5,"sRow":0,"eFile":4,"eRow":1,"mPiece":""}],"pgnMove":"Kxe2"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":7,"sRow":2,"eFile":6,"eRow":1,"mPiece":""}],"pgnMove":"Qg2+"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":5,"sRow":7,"eFile":5,"eRow":1,"mPiece":""}],"pgnMove":"Rf2"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":6,"sRow":1,"eFile":4,"eRow":3,"mPiece":""}],"pgnMove":"Qe4+"},"moveStrength":0,"PosEvaluation":0,"tags":[]},{"comment":"Comment","arrows":[],"move":{"movements":[{"sFile":4,"sRow":1,"eFile":3,"eRow":1,"mPiece":""}],"pgnMove":"Kd2"},"moveStrength":0,"PosEvaluation":0,"tags":[]}]}
```

**Comment by chichha** [comment]
Done, remove .lower() in parser.py

**Comment by lucalaissue** [comment]
fixed with #34
